src/test/java/org/opentripplanner/routing/algorithm/transferoptimization/model/passthrough/WalkDurationForStopCombinations.java

```diff
@@ -0,0 +1,60 @@
+package org.opentripplanner.routing.algorithm.transferoptimization.model.passthrough;
+
+import java.util.BitSet;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import org.opentripplanner.framework.lang.IntUtils;
+import org.opentripplanner.raptor.api.request.PassThroughPoint;
+
+/**
+ * This class is used to adjust the walk time - giving each path a unique generalized-cost. We want
```

Comment:
an unique*

Reply:
Actually, 'unique' is pronounced with a 'you'-sound so 'a' is correct.